Calculator/StringCalculator.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Calculator.Constants;
+using Calculator.Factories;
+using Calculator.Services.Numbers;
+
+namespace Calculator
+{
+    public class StringCalculator
+    {
+        private readonly INumberServiceFactory _numberServiceFactory;
+        private INumberService _numberService;
```

Comment:
Not a lekker idea - keeping state like this becomes weird depending on the lifestyle of this class (if it were a dependency in DI) and in things like concurrency. There's no intrinsic benefit to having this as a global variable, rather manage it locally for each method. Happy to chat more about why if needs be.